src/main/java/com/gotocompany/firehose/config/GrpcSinkConfig.java

```diff
@@ -17,4 +17,12 @@ public interface GrpcSinkConfig extends AppConfig {
     @Config.Key("SINK_GRPC_RESPONSE_SCHEMA_PROTO_CLASS")
     String getSinkGrpcResponseSchemaProtoClass();
 
+    @Config.Key("SINK_GRPC_ARG_KEEPALIVE_TIME_MS")
+    Integer getSinkGrpcArgKeepaliveTimeMS();
+
+    @Config.Key("SINK_GRPC_ARG_KEEPALIVE_TIMEOUT_MS")
+    Integer getSinkGrpcArgKeepaliveTimeoutMS();
```

Comment:
let's make this also to Long and default to 20 seconds.

Reply:
made changes